### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by nikoandpiko**
Fixes various grammar, typos, and updates YK image link to display properly.

Additionally cleaning up markdown syntax if desired.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,67 +10,77 @@ This is the development home for Cyberduck, a libre file transfer client for mac
...
 
-Run `mvn verify -DskipTests -DskipSign` to build without running any tests and skip codesign. Find build artifacts in 
- * `osx/target/Cyberduck.app`
- * `windows/target/Cyberduck.exe`
+Run `mvn verify -DskipTests -DskipSign` to build without running any tests and skip codesign. Find build artifacts in
+
+* `osx/target/Cyberduck.app`
+* `windows/target/Cyberduck.exe`
+
+Run with `-pyinstaller` to build installer packages with build artifacts
```
Comment (dkocher) [inline README.md:82]: There is a new typo introduced.

File: README.md [hunk 2]
```diff
@@ -10,67 +10,77 @@ This is the development home for Cyberduck, a libre file transfer client for mac
...
-These are nightly snapshot builds from the current development trunk featuring the latest bugfixes and enhancements. Be warned, tough, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
+
+These are nightly snapshot builds from the current development trunk featuring the latest bug fixes and enhancements. Be warned, though, these builds are potentially unstable and experimental. You can also switch to beta or snapshot builds in _Preferences → Update.
 
 ## Prerequisites
 
-- Java 11 SDK or later
-- Apache Ant 1.10.1 or later
-- Apache Maven 3.5 or later
+* Java 11 SDK or later
```
Comment (dkocher) [inline README.md:41]: Why do you consider dashes deprecated syntax?

**Review by dkocher — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (nikoandpiko) [reply README.md]: I was not familiar. It has now been reverted.

File: README.md (hunk 2 above)
Comment (nikoandpiko) [reply README.md]: Following [this standard](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md004), but it is only a suggestion. It's now been reverted.

**Review by dkocher — CHANGES_REQUESTED** [review]
Looks good to me. Please sqash the commits to merge.

**Comment by nikoandpiko** [comment]
@dkocher Any more changes requested? Merging is blocked until approved.

**Comment by dkocher** [comment]
> @dkocher Any more changes requested? Merging is blocked until approved.

Please sqash the commits.

**Comment by nikoandpiko** [comment]
Apologies. Need to fix this -- ran into issues with git.

**Review by dkocher — APPROVED** [review]

**Pull request merged**